interface/src/app/greenelab/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module includes a few utility components that will be shared in other
+ * modules to avoid duplicate codes.
+ */
+
+angular.module('greenelab.utils', [])
+.constant('ApiBasePath', '/api/v0/')
```

Comment:
Hmmm - I agree with this - ``greenelab.utils.js`` will probably get installed onto other servers of ours, so I agree that we should have this as a constant, but totally, maybe in a app/constants.js file?